app/controllers/course/assessment/question/scribing_controller.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+# rubocop:disable Metrics/ClassLength
+class Course::Assessment::Question::ScribingController < \
+  Course::Assessment::QuestionsController
+  load_and_authorize_resource :scribing_question,
+                              class: Course::Assessment::Question::Scribing,
+                              through: :assessment, parent: false
+
+  include CarrierWave::MiniMagick
+
+  def new
+    respond_to do |format|
+      format.html { render 'new' }
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.json { render_scribing_question_json }
+    end
+  end
+
+  def create # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+    # special handling for PDF files
+    if params&.[](:question_scribing)&.[](:file)&.content_type&.downcase == 'application/pdf'
+      file = params[:question_scribing][:file]
+      respond_to do |format|
+        if generate_pdf_questions(file)
+          format.json { render_success_json t('.success') }
+        else
+          format.json { render_failure_json t('.failure') }
+        end
+      end
+    else
+      respond_to do |format|
+        if @scribing_question.save
+          format.json { render_scribing_question_json }
+        else
+          format.json { render_failure_json t('.failure') }
+        end
+      end
+    end
+  end
+
+  def edit
+    respond_to do |format|
+      format.html { render 'edit' }
+      format.json { render_scribing_question_json }
+    end
+  end
+
+  def update
+    # Don't allow the user to edit attachment at the moment
```

Comment:
Can put this as a TODO instead. If need be, you can also guard against this in params by detecting the controller action. , but I think this would suffice for now.